java-api.md

```diff
@@ -199,6 +199,11 @@ Second generator would generate strings of length 8 from specified range of char
 "EF893232", "2E49D0AB", "BE129E15", "938FFC1C", "BB8A43ED", "829D1CA2", ...
 ```
 
+## Now methods
+
+These methods return current time, there are three variations: `nowDate()`, `nowLocalDate()` and `nowLocalDateTime()`.
```

Comment:
I agree that we can have also now() which returns millis in UTC, but I would not remove other options, it can be useful for unit/integration testing if someone has a domain class with Date / LocalDate / LocalDateTime field.